recipes/scdoc/all/test_package/conanfile.py

```diff
@@ -0,0 +1,9 @@
+from conans import ConanFile, Meson, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+
+    def test(self):
+        self.run(
+            f"scdoc < {os.path.join(self.source_folder,'test_package.1.scd')}", run_environment=True)
```

Comment:
```suggestion
         if not tools.cross_building(self):
            self.run(
                f"scdoc < {os.path.join(self.source_folder,'test_package.1.scd')}", run_environment=True)
```

Don't need to run when cross compiled, the binary won't run.

Reply:
corrected